company-coq.el

```diff
@@ -2174,9 +2174,10 @@ to a non-existent file (for an example of such a case, try
 \[Locate Library Peano] in 8.4pl3)."
   (if (and (equal lib-path "")
            (or (equal mod-name "Top")
-               (and (buffer-file-name proof-script-buffer)
-                    (equal (concat mod-name ".v")
-                           (file-name-nondirectory (buffer-file-name proof-script-buffer))))))
+               (and proof-script-buffer
+                    (and (buffer-file-name proof-script-buffer)
```

Comment:
Perfect. Since `and` can take any number of arguments there's no need to nest though, you can have just one `and` call.

Reply:
is it guaranteed that the conditions will be checked from left to right and it will stop if the first one is false?

Reply:
Thankfully yes